docs/connectors/additional-built-in-plugins.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Add Built-in Connectors"
```

Comment:
Other pages are named using the infinitive verb forms (installing, referencing, building...) so my suggestion is to use this form here as well.

```suggestion
title: "Adding built-in Connectors"
```